docs: update `nuxt.config` introduction (#2039)
nuxt/framework: docs/scripts/nuxt.config.md

# Nuxt configuration file

> Nuxt can be configured easily with one single file, called `nuxt.config`, with either a `.js`, `.ts` or `.mjs` extension.

Reviewer: minor rewording suggestion:

Nuxt can be easily configured with a single file called `nuxt.config`, which can either have a `.js`, `.ts` or `.mjs` extension.

> Nuxt can be configured easily with one single file, called `nuxt.config`, with either a `.js`, `.ts` or `.mjs` extension.

Reviewer: minor rewording suggestion:

Nuxt can be easily configured with a single file called `nuxt.config`, which can either have a `.js`, `.ts` or `.mjs` extension.

Author: ```suggestion
Nuxt can be easily configured with a single `nuxt.config` file, which can have either a `.js`, `.ts` or `.mjs` extension.
```
